ufo/user_test.py

```diff
@@ -0,0 +1,84 @@
+"""Test user module functionality."""
+from mock import MagicMock
+from mock import patch
+import os
+
+import flask
+from flask.ext.testing import TestCase
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session
+from sqlalchemy.orm import sessionmaker
+import unittest
+
+from . import app, db
```

Comment:
can we break up the import here into 2 lines?

from . import app
from . import db